spec/services/coordination/document_version_cache_spec.rb

```diff
@@ -0,0 +1,7 @@
+RSpec.describe Coordination::DocumentVersionCache do
+  subject(:document_version_cache) { described_class.new("content-id", payload_version:) }
+
+  let(:payload_version) { 1234 }
+
+  pending
```

Comment:
I didn't realised `pending` could be applied to the whole class. That's good to know.